contrib_bots/lib/converter.py

```diff
@@ -0,0 +1,235 @@
+         'cubic-decimeter': [0, 0.001, 'cubic-meter'],
+         'cubic-foot': [0, 0.028316846592, 'cubic-meter'],
+         'cubic-inch': [0, 0.000016387064, 'cubic-meter'],
+         'cubic-meter': [0, 1, 'cubic-meter'],
+         'cubic-yard': [0, 0.764554857984, 'cubic-meter'],
+         'fahrenheit': [255.372222, 0.555555, 'kelvin'],
+         'fluid-ounce': [0, 0.000029574, 'cubic-meter'],
+         'foot': [0, 0.3048, 'meter'],
+         'gram': [0, 1, 'gram'],
+         'hectare': [0, 10000, 'square-meter'],
+         'inch': [0, 0.0254, 'meter'],
+         'kelvin': [0, 1, 'kelvin'],
+         'liter': [0, 0.001, 'cubic-meter'],
+         'meter': [0, 1, 'meter'],
+         'mile': [0, 1609.344, 'meter'],
+         'nautical-mile': [0, 1852, 'meter'],
+         'ounce': [0, 28.349523125, 'gram'],
+         'pound': [0, 453.59237, 'gram'],
+         'square-centimeter': [0, 0.0001, 'square-meter'],
+         'square-foot': [0, 0.09290304, 'square-meter'],
+         'square-inch': [0, 0.00064516, 'square-meter'],
+         'square-kilometer': [0, 1000000, 'square-meter'],
+         'square-meter': [0, 1, 'square-meter'],
+         'square-mile': [0, 2589988.110336, 'square-meter'],
+         'square-yard': [0, 0.83612736, 'square-meter'],
+         'ton': [0, 1000000, 'gram'],
+         'yard': [0, 0.9144, 'meter']}
+
+PREFIXES = {'atto': -18,
+            'femto': -15,
+            'pico': -12,
+            'nano': -9,
+            'micro': -6,
+            'milli': -3,
+            'centi': -2,
+            'deci': -1,
+            'deca': 1,
+            'hecto': 2,
+            'kilo': 3,
+            'mega': 6,
+            'giga': 9,
+            'tera': 12,
+            'peta': 15,
+            'exa': 18}
+
+ALIASES = {'c': 'celsius',
+           'cm^3': 'cubic-centimeter',
+           'ft^3': 'cubic-foot',
+           'm^3': 'cubic-meter',
+           'y^3': 'cubic-yard',
+           'f': 'fahrenheit',
+           'ft': 'foot',
+           'g': 'gram',
+           'ha': 'hectare',
+           'in': 'inch',
+           'k': 'kelvin',
+           'l': 'liter',
+           'm': 'meter',
+           'mi': 'mile',
+           'nmi': 'nautical-mile',
+           'oz': 'ounce',
+           'lb': 'pound',
+           'cm^2': 'square-centimeter',
+           'ft^2': 'square-foot',
+           'in^2': 'square-inch',
+           'km^2': 'square-kilometer',
+           'm^2': 'square-meter',
+           'mi^2': 'square-mile',
+           'y^2': 'square-yard',
+           't': 'ton',
+           'y': 'yard'}
+
+HELP_MESSAGE = ('Converter usage:\n'
+                '`@convert <number> <unit_from> <unit_to>`\n'
+                'Converts `number` in the unit `unit_from` to '
+                'the `unit_to` and prints the result\n'
+                '`number`: floating point number, e.g. 12, 13.05, 0.002\n'
+                '`unit_from` and `unit_to` are two of the following units:\n'
+                '* Bit, byte\n'
```

Comment:
This list is a bit different from the docs and the variables - e.g. temperature abbreviations are capitalized. Could you make sure those are matching?